decentraland/unity-renderer
# Feat: changed main camera from overlay to base

## What does this PR change?

Fix #3583 
Changes main camera from overlay to base and moves infinite floor slightly below ground to avoid compenetrations

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/main-camera-base
2. Verify that everything is rendered regularly
3. Verify specifically that transparent materials and infinite floor are correctly rendered

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
